manifest_staging/charts/gatekeeper/templates/gatekeeper-controller-manager-deployment.yaml

```diff
@@ -63,8 +63,8 @@ spec:
         - --prometheus-port={{ .Values.controllerManager.metricsPort }}
         - --logtostderr
         - --log-denies={{ .Values.logDenies }}
-        - --emit-admission-events={{ .Values.emitAdmissionEvents }}
-        - --log-stats-admission={{ .Values.logStatsAdmission }}
+        {{ if hasKey Values "emitAdmissionEvents" }}- --emit-admission-events={{ .Values.emitAdmissionEvents }}{{- end }}
+        {{ if hasKey Values "logStatsAdmission" }}- --log-stats-admission={{ .Values.logStatsAdmission }}{{- end }}
```

Comment:
These flags are in alpha for a long time and has been set for many GK versions. I can remove these changes if this is a breaking concern.